### ISSUE TITLE
Panic in latest release

### ISSUE THREAD

**Issue by angadn** [nishanths/exhaustive#10]
I get this on running the command
```
github.com/nishanths/exhaustive.missingCasesTextEdit(0xc000135b00, 0xc00a4ef600, 0xc00a846500, 0xc00a84cc30, 0xc00a8451c0, 0xc004ffc0a0, 0xc0275cf800, 0x0, 0x0, 0x0, ...)
        /Users/angadn/go/pkg/mod/github.com/nishanths/exhaustive@v0.2.1/switch.go:403 +0x891
github.com/nishanths/exhaustive.computeFix(0xc01f0e4000, 0xc000135b00, 0xc00a4ef600, 0xc00a84cc30, 0xc00a8451c0, 0xc004ffc0a0, 0xc024121800, 0xc0275cf800, 0x0, 0x0, ...)
        /Users/angadn/go/pkg/mod/github.com/nishanths/exhaustive@v0.2.1/switch.go:267 +0xe5
github.com/nishanths/exhaustive.reportSwitch(0xc01f0e4000, 0xc00a84cc30, 0xc00a8451c0, 0x0, 0xc004ffc0a0, 0xc027542900, 0xc0275cf800, 0xc00a4ef600)
        /Users/angadn/go/pkg/mod/github.com/nishanths/exhaustive@v0.2.1/switch.go:241 +0xff
github.com/nishanths/exhaustive.checkSwitchStatements_.func1(0x13aaeb8, 0xc00a84cc30, 0x1, 0xc0374d4200, 0x6, 0x20, 0x1)
```

**Comment by nishanths** [comment]
Thank you for the report. I'm looking.

**Comment by nishanths** [comment]
This should be fixed from v0.2.2 onward. Thanks again for reporting 🙂

**Comment by angadn** [comment]
@nishanths Thank you for maintaining this project and fixing this so promptly!
